web/frontend/src/components/slam-2d-render.vue

```diff
@@ -8,6 +8,36 @@ import { MapControls } from 'three/examples/jsm/controls/OrbitControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
 
+/*
+ * // Leaving additional color map commented for if we want to change to a different scheme.
+ * // const colorMapViridis = [
+ * //   [253, 231, 37],
+ * //   [181, 222, 43],
+ * //   [110, 206, 88],
+ * //   [53, 183, 121],
+ * //   [31, 158, 137],
+ * //   [38, 130, 142],
+ * //   [49, 104, 142],
+ * //   [62, 73, 137],
+ * //   [72, 40, 120],
+ * //   [68,1,84]
+ * // ]
+ */
+
+// this color map is greyscale
+const colorMapGrey = [
+  [247, 247, 247],
+  [239, 239, 239],
+  [223, 223, 223],
+  [202, 202, 202],
+  [168, 168, 168],
+  [135, 135, 135],
+  [109, 109, 109],
+  [95, 95, 95],
+  [74, 74, 74],
+  [0, 0, 0],
+];
```

Comment:
Can we try creating the vector3s once rather than doing so for every point?

I think doing so will improve performance.

```suggestion
].map(([red, green, blue]) =>
    new THREE.Vector3(red, green, blue).multiplyScalar(1 / 255));
};
```

Reply:
didnt know i could just slap this on at the start, thanks!